webhooks-with-pgstream.mdx

```diff
@@ -0,0 +1,555 @@
+---
+title: 'Postgres webhooks with pgstream'
+description: 'A simple tutorial for calling webhooks on Postgres data and schema changes using pgstream.'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/pgstream/introducing-pgstream@2x.jpg
+  alt: pgstream
+author: Tudor Golubenco
+authorEmail: tudor@xata.io
+date: 08-27-2024
+tags: ['postgres', 'fpPgstream', 'oss', 'open-source']
+published: true
+slug: postgres-webhooks-with-pgstream
+ogImage: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/pgstream/introducing-pgstream@2x.jpg
+---
+
+[pgstream](https://github.com/xataio/pgstream) is a CDC (Change-Data-Capture) tool focused on PostgreSQL. Among other [things](https://github.com/xataio/pgstream?tab=readme-ov-file#features), it can be used to call webhooks whenever there is a data (or schema) change in a Postgres database. This means that whenever a row is inserted, updated, or deleted, a webhook is called with the data from the row.
```

Comment:
```suggestion
[pgstream](https://github.com/xataio/pgstream) is a CDC (Change-Data-Capture) tool focused on PostgreSQL. Among other [things](https://github.com/xataio/pgstream?tab=readme-ov-file#features), it can be used to call webhooks whenever there is a data (or schema) change in a Postgres database. This means that whenever a row is inserted, updated, or deleted, or a table is created, altered, truncated or deleted, a webhook is notified of the relevant event details.
```